tests/test_importexport.py

```diff
@@ -2265,6 +2265,20 @@ def test_bench_with_same_io_name(self):
         self.check_io(pyrtl.Input, ["G1", "G2", "G3"])
         self.check_io(pyrtl.Output, ["tmp3", "G4"])
 
+class TestOutputVerilog(unittest.TestCase):
+    def test_custom_module_name(self):
+        import io, pyrtl
+        pyrtl.reset_working_block()
+        a, b = pyrtl.Input(1, 'a'), pyrtl.Input(1, 'b')
+        out = pyrtl.Output(name='out')
+        out <<= a & b
+
+        buf = io.StringIO()
+        pyrtl.output_to_verilog(buf, module_name='custom_top')
```

Comment:
We will also need a similar test for `output_verilog_testbench` with a custom `module_name`